plugins/out_s3/s3.c

```diff
@@ -700,7 +700,7 @@ static int cb_s3_init(struct flb_output_instance *ins,
          */
         ctx->upload_chunk_size = ctx->file_size;
         if (ctx->file_size > MAX_FILE_SIZE_PUT_OBJECT) {
-            flb_plg_error(ctx->ins, "Max total_file_size is 50M when use_put_object is enabled");
+            flb_plg_error(ctx->ins, "Max total_file_size is 1G when use_put_object is enabled");
```

Comment:
Thanks!!

The best practice would have been if we had originally done somehting like:

```
#define MAX_FILE_SIZE_PUT_OBJECT             1000000000 
#define MAX_FILE_SIZE_PUT_OBJECT_STR    1G 
```

And then use STR in the mesg. Can you do that

Reply:
Done